apps/namadillo/src/App/Ibc/ShieldAllAssetList.tsx

```diff
@@ -0,0 +1,65 @@
+import { Asset } from "@chain-registry/types";
+import { Checkbox } from "@namada/components";
+import { TokenCurrency } from "App/Common/TokenCurrency";
+import { AssetWithBalance } from "atoms/integrations";
+import BigNumber from "bignumber.js";
+import clsx from "clsx";
+import { getAssetImageUrl } from "integrations/utils";
+import { toDisplayAmount } from "utils";
+
+type AssetWithBalanceMap = Record<string, AssetWithBalance>;
```

Comment:
Maybe we could reuse this type since we also have `Record<string, AssetWithBalance>` in other places.